src/components/exchange/ExchangeNativeField.js

```diff
@@ -67,6 +75,8 @@ const ExchangeNativeField = (
     return colors.alpha(color, opacity);
   }, [colors, isFocused, nativeAmount]);
 
+  const isInputFocused = ref?.current?.isFocused();
```

Comment:
we talked about this previously but actually, this component has an internal state `isFocused` that we can use instead

Reply:
yes sir